www/assets/js/components/SearchPage.test.tsx

```diff
@@ -49,8 +49,6 @@ jest.mock("../lib/api", () => ({
 }))
 const spySearch = jest.mocked(search)
 
-jest.mock("lodash.debounce", () => jest.fn(fn => fn))
```

Comment:
This was the only usage of `_.debounce` or `lodash.debounce` in our source code. Removing this allows us to remove the unused dependency.